edifikana/back-end/build.gradle.kts

```diff
@@ -15,4 +15,7 @@ dependencies {
     implementation ("com.google.auth:google-auth-library-credentials:1.23.0")
     implementation ("com.google.apis:google-api-services-drive:v3-rev20220815-2.0.0")
     implementation ("com.google.apis:google-api-services-sheets:v4-rev20240416-2.0.0")
+    implementation ("com.google.firebase:firebase-firestore:24.11.1")
+    implementation ("com.google.firebase:firebase-admin:9.2.0")
+    implementation ("com.google.firebase:firebase-common-ktx:21.0.0")
```

Comment:
In the future we will want to move this version numbers out of this file and into the [versions](https://github.com/CodeHavenX/MonoRepo/blob/main/versions.properties) file. Just an FYI, no action is needed right now.